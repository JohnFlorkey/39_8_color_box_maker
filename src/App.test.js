import { render } from '@testing-library/react';
import App from './App';

// smoke test
test('renders learn react link', () => {
  render(<App />);
});

// snapshot test
it('matches snapshot', () => {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
