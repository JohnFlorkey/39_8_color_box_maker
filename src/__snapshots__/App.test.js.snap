// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Color Box Maker
    </h1>
    <div>
      <div
        class="BoxList"
      />
      <form
        class="NewBoxForm"
      >
        <label
          class="NewBoxForm-label"
          for="backgroundColor"
        >
          Background Color
        </label>
        <input
          class="NewBoxForm-input"
          id="backgroundColor"
          name="backgroundColor"
          type="text"
          value=""
        />
        <label
          class="NewBoxForm-label"
          for="width"
        >
          Width
        </label>
        <input
          class="NewBoxForm-input"
          id="width"
          name="width"
          type="text"
          value=""
        />
        <label
          class="NewBoxForm-label"
          for="height"
        >
          Height
        </label>
        <input
          class="NewBoxForm-input"
          id="height"
          name="height"
          type="text"
          value=""
        />
        <button>
          Add Box
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
